tools/flang2/flang2exe/outliner.cpp

```diff
@@ -1430,7 +1409,7 @@ static SPTR
 member_with_offset(SPTR member, int offset)
 {
   for( ; member > NOSYM && ADDRESSG(member) < offset; member = SYMLKG(member)) {
-    if (ADDRESSG(member) = offset)
+    if (ADDRESSG(member) == offset)
```

Comment:
Will this ever be true?

Reply:
Thanks for catching this. IIUC, the intention of the code (introduced in fdd777cee941eb7df368a54abad2de4dff744194) is to create a dedicated name table entry for `addr`, in order to make the dumps easier to read (otherwise there would be many stores associated with the same NME, which would be confusing). I believe it tries to reuse an existing NME if one already exists for the given `offset`. However, the loop in `member_with_offset` is clearly buggy. Let me try to fix this and perhaps @mleair or someone at NVIDIA could review my attempt.

Reply:
BTW, `XBIT(69, 0x80000)` is never enabled by default (and is poorly documented), so this buggy code has never caused any issue.

Reply:
Create #1122 to address this issue. For this PR, I have modified the code to silence the warning in `member_with_offset` (NFC).